examples/animation/multiple_axes.py

```diff
@@ -0,0 +1,79 @@
+"""
+=======================
+Multiple axes animation
+=======================
+
+This example showcases:
+
+- how animation on subplots works,
+- using a figure artist in the animation.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.animation as animation
+from matplotlib.patches import ConnectionPatch
+
+fig, (axl, axr) = plt.subplots(
+    ncols=2,
+    sharey=True,
+    figsize=(6, 2),
+    gridspec_kw=dict(width_ratios=[1, 3], wspace=0),
+)
+axl.set_aspect(1)
+axr.set_box_aspect(1 / 3)
+axr.yaxis.set_visible(False)
+axr.xaxis.set_ticks([0, np.pi, 2 * np.pi], ["0", r"$\pi$", r"$2\pi$"])
+
+# draw circle with initial point in left Axes
+x = np.linspace(0, 2 * np.pi, 50)
+axl.plot(np.cos(x), np.sin(x), "k", lw=0.3)
+(point,) = axl.plot(0, 0, "o")
```

Comment:
Typically, we only use the comma, which is enough to make it a tuple.

```suggestion
point, = axl.plot(0, 0, "o")
```

see e.g. https://matplotlib.org/devdocs/gallery/animation/simple_anim.html#sphx-glr-gallery-animation-simple-anim-py

I'm a bit torn on this. On the one hand, I wouldn't use the extra parentheses in production code. On the other hand, I can see an argument that only-comma is subtle and one wants to be more on-the-nose in an example. But if that's the case, we should have a general discussion on the topic and use it consistently throughout the examples. For this PR I suggest only to use comma. Also for `sine`

Reply:
Fixed.  
I originally used `point, ` without parentheses but `black` added them without me noticing it (I ran `black` to wrap the long lines nicely for me, but it changed some other things too)